MANIFEST.in

```diff
@@ -15,11 +15,9 @@ prune docs/_build
 prune dist
 prune examples
 
-prune .*
-prune .git
-prune .tox
-
-prune nengo_spa/examples/.ipynb_checkpoints
+# Patterns to exclude from any directory
+global-exclude *.git*
+global-exclude *-checkpoint.ipynb
```

Comment:
Did you test this change?
Why did you remove `prune .*` (Though maybe it wasn't working because `.git` and `.tox` are explicitly listed?)
We probably should keep `prune .tox`.

Reply:
My intention was to copy [core nengo's `MANIFEST.in`](https://github.com/nengo/nengo/blob/master/MANIFEST.in), but I might have done this incorrectly. Would you mind explaining to me the differences between this project's `MANIFEST.in` and core nengo's?

Reply:
The differences are probably not very meaningful. There are multiple ways to achieve the same result. I suppose I was using `prune` for `.git` and `.tox` because those directories should only appear in the root, but `global-exclude` might work too (I'm just not entirely sure whether it excludes directories or only files, that's why I was asking whether this change was tested).

There is a `global-exclude *.tox*` in the Nengo `MANIFEST.in` and even though we are removing the tox configuration some people might still use it or have an old `.tox` directory lieing around that should not be committed.

Reply:
Tested it and added the exclusion of `tox`. Looked good from visual inspection.